apps/condo/cypress/e2e/pages/property.cy.js

```diff
@@ -1,72 +1,88 @@
-            cy.task('keystone:createUserWithProperty', true).then((response) => {
+        it('can create properties and manipulate their maps', () => {
+            const trace = new SimpleTracer('user.canCreatePropertyMap', 'property')
+            const spanPrepare = trace.startSpan('1.createUserWithOrganization')
+            cy.task('keystone:createUserWithOrganization').then(async (response) => {
                 authUserWithCookies(response)
+                spanPrepare.finish()
 
-                const propertyMapCreate = new PropertyMapCreate()
-                propertyMapCreate
-                    .visit()
-                    .clickOnPropertyTableRow()
-                    .clickEditPropertyMapButton()
-                    .clickSection()
-                    .clickRemoveSection()
-                    .clickOnEditMenu()
-                    .clickEditSection()
-                    .typeFloorCount()
-                    .typeUnitsOnFloorCount()
-                    .clickSubmitButton()
-                    .clickSavePropertyMap()
-            })
-        })
+                const organization = response.organization
 
-        it('can create and copy section', () => {
-            cy.task('keystone:createUserWithProperty', true).then((response) => {
-                authUserWithCookies(response)
+                const condo = new Condo()
+                condo.visit()
 
-                const propertyMapEdit = new PropertyMapEdit()
-                propertyMapEdit
-                    .visit()
-                    .clickOnPropertyTableRow()
-                    .clickEditPropertyMapButton()
-                    .cleanUp()
-                    .createTestSection()
-                    .clickOnEditMenu()
-                    .clickEditSection()
-                    .clickSectionEditMode()
-                    .clickSubmitButton()
-                    .clickSavePropertyMap()
-            })
-        })
+                let activeSpan = trace.startSpan('2.createPropertyMap')
+                cy.task('keystone:createProperty', organization).then(() => {
+                    condo.clickOnMenuItem('property')
 
-        it('can add, remove and update section unit', () => {
-            cy.task('keystone:createUserWithProperty', true).then((response) => {
-                authUserWithCookies(response)
+                    const propertyMapCreate = new PropertyMapCreate()
+                    propertyMapCreate
+                        .clickOnPropertyTableRow()
+                        .clickEditPropertyMapButton()
+                        .clickSection()
+                        .clickRemoveSection()
+                        .clickOnEditMenu()
+                        .clickEditSection()
+                        .typeFloorCount()
+                        .typeUnitsOnFloorCount()
+                        .clickSubmitButton()
+                        .clickSavePropertyMap()
+                })
+                activeSpan.finish()
```

Comment:
it looks strange. We are trying to calculate not the test time, but the test initialization time.